examples/meta/src/observers/gaussian_mixture_models.sg

```diff
@@ -0,0 +1,40 @@
+File f_feats_train = csv_file("../../data/classifier_4class_2d_linear_features_train.dat")
+
+Math:init_random(1)
+
+#![create_features]
+Features features_train = features(f_feats_train)
+#![create_features]
+
+#![create_gmm_instance]
+int num_components = 3
+GMM gmm(num_components)
+#![create_gmm_instance]
+
+#![add_observer]
+ParameterObserver obs = parameter_observer("ParameterObserverLogger")
```

Comment:
as said in my other unrelated comment, it would be good/better if this guy used the mechanics as used in `SGObject::to_string()`

Reply:
I may update it in another PR since I do not want to bloat this one :)